src/install/unix.rst

```diff
@@ -230,9 +234,55 @@ CouchDB no longer ships with any daemonization scripts.
 The couchdb team recommends `runit <http://smarden.org/runit/>`_ to
 run CouchDB persistently and reliably. Configuration of runit is
 straightforward; if you have questions, reach out to the CouchDB
```

Comment:
please make the mailing list reference a link, like you've already done for runit. It might be worth doing this for IRC  channel too?